src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -1953,6 +2282,246 @@
+            "ERROR"
+          ]
+        },
+        "href": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string",
+          "description": "The unique identifier of the Request"
+        },
+        "lastModified": {
+          "type": "string",
+          "description": "The date when the Request was last modified"
+        },
+        "mainDestination": {
+          "description": "The main destination of the Request",
+          "$ref": "#/definitions/Location"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the Request"
+        },
+        "operations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Link"
+          }
+        },
+        "owner": {
+          "description": "The employee who owns the Request",
+          "$ref": "#/definitions/Employee"
+        },
+        "pendingApproval": {
+          "type": "boolean",
+          "description": "Indicates whether this Request is pending approval"
+        },
+        "policy": {
+          "description": "The policy that applies to the Request",
+          "$ref": "#/definitions/ResourceLink"
+        },
+        "requestId": {
+          "type": "string",
+          "description": "The public key of the Request (unique per customer)"
+        },
+        "startDate": {
+          "type": "string",
+          "description": "The start date of the Request (in the format YYYY-MM-DD)"
+        },
+        "startTime": {
+          "type": "string",
+          "description": "The start time of the Request (in the format HH:MM)"
+        },
+        "submitDate": {
+          "type": "string",
+          "description": "The date when the Request was submitted (last submit date in case of recall)"
+        },
+        "totalApprovedAmount": {
+          "description": "The total amount of approved expenses in the Request, expressed in the reimbursement currency of the employee at the time he created the Request",
+          "$ref": "#/definitions/Amount"
+        },
+        "totalPostedAmount": {
+          "description": "The total amount of the Request, expressed in the reimbursement currency of the employee at the time he created the Request",
+          "$ref": "#/definitions/Amount"
+        },
+        "totalRemainingAmount": {
+          "description": "The total amount not included in an Expense report, expressed in the reimbursement currency of the employee at the time he created the Request",
+          "$ref": "#/definitions/Amount"
+        },
+        "travelAgency": {
+          "description": "The travel agency office that is managing the trip associated to this Request",
+          "$ref": "#/definitions/ResourceLink"
+        },
+        "type": {
+          "description": "The type of the Request, inherited from the Request policy type",
+          "$ref": "#/definitions/RequestType"
+        }
+      }
+    },
+    "RequestDetails": {
```

Comment:
Request and RequestDetails look like duplicates- is there a difference between the two?

Reply:
RequestDetails is a response template. Request is a request body.